fastlane/Fastfile

```
@@ -293,6 +292,21 @@ platform :ios do
 
   desc "Install CocoaPods dependencies"
   lane :install_pods do
+
+    UI.message("Modifying objectVersion in project.pbxproj due to a bug in Cocoapods...")
```

rubocop

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)